style.css

```diff
@@ -798,6 +1006,8 @@ h3 {
   }
 }
 
+/* modal window desktop */
+

```

Comment:
- [ ]  As stated in the project requirement Kindly make sure that the popup is responsive, it fits mobile and desktop sizes properly.So Kindly fix the responsiveness for the mobile screen size.
- Your Project Pop-up for mobile screen size Look likes this : 

![Capture](https://user-images.githubusercontent.com/93598189/209362774-e90a7669-c703-4bd6-940c-8e2a9b40810e.PNG)

-  The Figma Pop-up for Mobile Screen size Look likes this : 

![Capture](https://user-images.githubusercontent.com/93598189/209363041-db49ccdd-9f8f-4d55-8db8-822fe5c4f26e.PNG)